referral/templates/referral/new-FQHC-referral.html

```diff
@@ -1,13 +1,16 @@
 {% extends "pttrack/base.html" %}
 
+{% load i18n %}
+<!-- trans finished -->
```

Comment:
What's the purpose of this comment?

Reply:
This can be deleted, it was just so that I'm sure that I've already finished the translation of that particular file.